attachment_synchronize/models/task.py

```diff
@@ -77,8 +77,9 @@ class StorageTask(models.Model):
         string="File Type",
         help="The file type determines an import method to be used "
              "to parse and transform data before their import in ERP")
-    active = fields.Boolean(default=True)
+    enabled = fields.Boolean('Enabled', default=True)
```

Comment:
because it will disappear from tasks tree view in storage_backend, and if deactivated in error it will not be shown anymore. That's the only reason reason

Reply:
I don't get it, disapear from task tree view? `attachment_ids` is not present in task view so the attachment should never appear in task view.
If unactive by error, it can be active again to correct the error
I mean, the purpose of active field is to archive stuff and so make it disappear, I think it is good to has it yes

Reply:
ok, I try to explain better. The `active` field is related to the task not to the `attachment_ids`. If u uncheck it, will be archived and it will not be shown in task tree view inside `storage_backend` form view.

Reply:
Oh yes, my bad, it is on the task, I was thinking of the attachment sorry!
You are right then, if unactive, it can't be active again from the UI, this may be an issue.
well, the enable field seems like a good idea then but such task should be ignored indeed

Sorry for the noise!